src/Mealz/MealBundle/Entity/Slot.php

```diff
@@ -128,4 +135,13 @@ public function setSlug(string $slug): string
     {
         return $this->slug = $slug;
     }
+
+    public function getParticipants(): ArrayCollection
```

Comment:
Could be an one-liner with ??-operator

Reply:
We are returning a copy of participant collection. ?? operator won't do that.

Reply:
Discussed.